ko/documentation/installation/index.md

```diff
@@ -338,6 +338,14 @@ RVM은 다른 gemset도 관리해 줍니다.
 macOS, Linux나 다른 UNIX-계열 운영체제에서만 사용 가능합니다.
 
 
+### RVM 4 Windows
+{: #rvm-windows}
```

Comment:
`RVM 4 Windows` should be added to TOC between "RVM" and "uru."

Reply:
My idea was to make it analogous to "rbenv for windows", which also does not have its own entry in TOC, but follows "rbenv". So does "RVM 4 Windows" follow "RVM".

Reply:
Oh, I understand.